nucypher/network/server.py

```diff
@@ -220,14 +226,16 @@ def reencrypt():
         # Deserialize and instantiate the request
         reenc_request = ReencryptionRequest.from_bytes(request.data)
 
-        # Deserialize and instantiate ConditionLingo from the request data
-        lingos = _deserialize_rust_lingos(reenc_request=reenc_request)
+        # obtain conditions from request
+        condition_lingo_list = json.loads(
+            str(reenc_request.conditions)
+        )  # Conditions -> str -> List[Lingo]
```

Comment:
Is `# Conditions -> str -> List[Lingo]` some sort of convention? For linter rules?

Reply:
Just clarification on terminology/expectation 😄 . Everything being called condition/lingo can be confusing.